entity-framework/core/modeling/shadow-properties.md

```diff
@@ -43,3 +43,12 @@ var blogs = context.Blogs
 ```
 
 Shadow properties cannot be accessed after a no-tracking query since the entities returned are not tracked by the change tracker.
+
+## Property bag entity types
+
+> [!NOTE]
+> Support for Property bag entity types was added in EF Core 5.0.
+
+Entity types that contain only indexer properties are known as property bag entity types. These entity types don't have shadow properties. Currently only `Dictionary<string, object>` is supported as a property bag entity type. This means that it must be configured as a shared entity type with a unique name and the corresponding `DbSet` property must be implemented using a `Set` call.
```

Comment:
Where do we document indexer properties?

Reply:
Same page

Reply:
https://github.com/dotnet/EntityFramework.Docs/issues/2018

Reply:
So you updated title without adding docs for indexer properties. I got confused in that.